### PR TITLE
OCI Integration

### LINKED ISSUE #2416: Add Support of OCI Vault into Helidon
This is the same topic as https://github.com/oracle/helidon/issues/2395 "Add Support of Hashicorp Vault into Helidon" but specifically the OCI Vault service and, like that ER, we are researching best approaches to this particularly as there are a number of solutions and approaches in this area.

### PR DESCRIPTION
**Pull request by tomas-langer**
OCI integration based on reactive `WebClient`.
Integrates OCI Object Storage, Vault (including secrets, encryption, signatures) and adding metrics (Telemetry).

Resolves #2416

### CODE REVIEW

File: docs/mp/graphql/01_mp_graphql.adoc [hunk 1]
```diff
@@ -145,7 +145,7 @@ and POJO's and you will be able to access these via the URL's described below.
 As per the instructions <<mp/introduction/02_microprofile.adoc, here>> ensure you have added a
 `src/main/resources/META-INF/beans.xml` file, so the CDI implementation can pick up your classes.
 
-A `Main` class is not needed, you can configure `io.helidon.microprofile.cdi.Main` as the entry point.
+A `VaultCdiMain` class is not needed, you can configure `io.helidon.microprofile.cdi.Main` as the entry point.
```
Comment (spericas) [inline docs/mp/graphql/01_mp_graphql.adoc:148]: `VaultCdiMain` class?

File: examples/integrations/oci/objectstorage-reactive/src/main/resources/logging.properties [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+# HelidonConsoleHandler uses a SimpleFormatter subclass that replaces "!thread!" with the current thread
+java.util.logging.SimpleFormatter.format=%1$tY.%1$tm.%1$td %1$tH:%1$tM:%1$tS %4$s %3$s !thread!: %5$s%6$s%n
+
+# Global logging level. Can be overridden by specific loggers
+.level=INFO
+
+io.helidon.level=INFO
+io.helidon.integrations.level=INFO
+io.helidon.webclient.level=FINEST
```
Comment (spericas) [inline examples/integrations/oci/objectstorage-reactive/src/main/resources/logging.properties:28]: FINEST?

**Review by spericas — APPROVED** [review]

File: docs/mp/graphql/01_mp_graphql.adoc (hunk 1 above)
Comment (tomas-langer) [reply docs/mp/graphql/01_mp_graphql.adoc]: Too eager refactoring of my IDE, fixed.

File: examples/integrations/oci/objectstorage-reactive/src/main/resources/logging.properties (hunk 2 above)
Comment (tomas-langer) [reply examples/integrations/oci/objectstorage-reactive/src/main/resources/logging.properties]: Fixed.

**Comment by tomas-langer** [comment]
Fixed and rebased on master.

**Review by Verdent — APPROVED** [review]
LGTM

**Pull request merged**